seacrowd/sea_datasets/tha_lotus/tha_lotus.py

```diff
@@ -0,0 +1,272 @@
+"""
+SEA Crowd Data Loader for Bloom Speech.
+"""
+import os
+from typing import Dict, List, Tuple
+
+import datasets
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import TASK_TO_SCHEMA, Licenses, Tasks
+
+import pandas as pd
+from collections import Counter
+from collections.abc import KeysView, Iterable
+
+_CITATION = r"""
+@INPROCEEDINGS{thaiLOTUSBN,
+  author={Chotimongkol, Ananlada and Saykhum, Kwanchiva and Chootrakool, Patcharika and Thatphithakkul, Nattanun and Wutiwiwatchai, Chai},
+  booktitle={2009 Oriental COCOSDA International Conference on Speech Database and Assessments}, 
+  title={LOTUS-BN: A Thai broadcast news corpus and its research applications}, 
+  year={2009},
+  volume={},
+  number={},
+  pages={44-50},
+  doi={10.1109/ICSDA.2009.5278377}}
+"""
+
+logger = datasets.logging.get_logger(__name__)
+
+_LOCAL = False
+_LANGUAGES = ["tha"]
+
+
+_DATASETNAME = "tha_lotus"
+_DESCRIPTION = r"""
+The Large vOcabualry Thai continUous Speech recognition (LOTUS) corpus was designed for developing large vocabulary
+continuous speech recognition (LVCSR), spoken dialogue system, speech dictation, broadcast news transcriber.
+It contains two datasets, one for training acoustic model, another for training a language model.
+"""
+
+_HOMEPAGE = "https://github.com/korakot/corpus/tree/main/LOTUS"
+_LICENSE = Licenses.CC.value
```

Comment:
Could you please change the license to Creative Commons Attribution Non Commercial Share Alike 3.0 (cc-by-nc-sa-3.0)?

Reply:
thanks @holylovenia!~ missed this detail